internals/scripts/extract-intl.js

```diff
@@ -14,14 +14,14 @@ const pkg = require('../../package.json');
 const presets = pkg.babel.presets;
 const plugins = pkg.babel.plugins || [];
 
-const i18n = require('../../app/i18n');
-import { DEFAULT_LOCALE } from '../../app/containers/App/constants';
-
 require('shelljs/global');
 
+import { DEFAULT_LOCALE } from '../../app/containers/App/constants';
+import { appLocales } from '../../app/i18n';
```

Comment:
I don't think `import`s work here?! This is ran in Node, untranspiled, which doesn't have `import`?